content/tutorials/marketplace-fees.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Fees
+description: The Ocean Protocol defines various fees for creating a sustainability loop.
+---
+
+## Path to sustainability
+
+Ocean Protocol aims to be a community-driven project. Through OceanDAO, the project aims at achieving decentralized governance and sustainability loop. For the project to grow and improve, the participants need to be rewarded for their contributions. The Ocean Protocol's smart contracts include a fee mechanism to maintain the sustainability loop.
+
+## Fee types
+
+### Swap fee
+
+Swap fees are collected whenever someone swaps a datatoken for basetoken (e.g., OCEAN) or basetoken for datatoken. The swap could be inside a pool (using an automated market maker) or in a fixed-rate exchange.
```

Comment:
I know "datatoken" is a single word for various reasons but @trentmc have we decided to do the same for "basetoken"? Or should we have two words here?

Reply:
I posed the same question in the core team Slack. The conclusion was that datatoken is 1 word, base token is two words.